README.md

```diff
@@ -502,6 +502,15 @@ const clean = sanitizeHtml('<p><iframe src="https://us02web.zoom.us/embed/12345"
 });
 ```
 
+### Script Filters
+
+Similarly to iframes you can allow a script tag on a list of whitelisted domains or hostnames
+
+```js
```

Comment:
Please include a full example like the one directly above. Similar to what you have in your tests.

Reply:
Thanks for the feedbacks I did the changes

Reply:
Thank you

Reply:
Let's change that language to "allow-listed."

Reply:
i'll take care of that.